web-services/deploy/application/src/main/wildfly/add-datawave-configuration.cli

```diff
@@ -202,7 +199,6 @@ jms-queue add --queue-address=QueryMetricsQueue --entries=queue/QueryMetrics
 /subsystem=ejb3/strict-max-bean-instance-pool=slsb-strict-max-pool/:write-attribute(name=timeout-unit,value=MILLISECONDS)
 /subsystem=ejb3:write-attribute(name=default-slsb-instance-pool,value=slsb-strict-max-pool)
 /subsystem=transactions/:write-attribute(name=default-timeout,value=${jboss.transaction.time.out})
-/subsystem=ee/managed-executor-service=default:write-attribute(name=max-threads,value=${jboss.managed.executor.service.default.max.threads})
 
```

Comment:
Is there a different way to set max-threads for the managed executor in W24? We've had deployment issues in the past when we've relied on whatever default gets set

Reply:
Added this line back in. At one point this line was causing trouble, but with this clean branch, that doesn't appear to be the case anymore.